Assets/Scripts/WeakPoint.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WeakPoint : MonoBehaviour
+{
+    public SphereCollider sphereCollider;
+    public float focusDownDur;
+    public float size;
+
+    private ParticleSystem particleSystem;
+
+    public bool active = false;
+    // Start is called before the first frame update
+    void Start()
+    {
+        EnemyAI enemyAI = GetComponentInParent<EnemyAI>();
+        particleSystem = GetComponent<ParticleSystem>();
+        
+        sphereCollider = gameObject.AddComponent(typeof(SphereCollider)) as SphereCollider;
+        sphereCollider.isTrigger = true;
+        sphereCollider.radius = enemyAI.weakPointSize;
+        focusDownDur = enemyAI.focusDownDurBase;
+        particleSystem.Stop();
```

Comment:
This statement is unnecessary if you uncheck play on awake in the particle system.